src/js/components/RadioButtonGroup/RadioButtonGroup.js

```diff
@@ -68,14 +62,6 @@ const RadioButtonGroup = forwardRef(
       return result;
     }, [options, value]);
 
-    useEffect(() => {
-      // if tab comes back to RadioButtonGroup when there still is no selection,
-      // we want focus to be on the first RadioButton
-      if (focus && !valueIndex) {
-        optionRefs.current[0].focus();
-      }
-    }, [focus, valueIndex]);
-
```

Comment:
Nice cleanup. Did it pass all the tests w/o it? Maybe we can try deploying it on the DS to ensure it works as part of an integral solution?

Reply:
I just tested using keyboard for a RadioButtonGroup on its own as well as a RadioButtonGroup in a FormField. The behavior seems to be working as expected for Chrome, Firefox, Safari, and with VoiceOver. I wish I could remember why in the initial PR I added the useEffect since it seems like my other addition to `focusable` logic was sufficient.

I am comfortable with removing the useEffect.

Reply:
Hey @ShimiSun! Good questions, just confirming I also tested the keyboard behavior across browsers and with a RadioButtonGroup within and without Form/FormField.

Reply:
Wonderful! Thank you ladies. Missed you - you two superwomen!